norddrop/src/device.rs

```diff
@@ -15,13 +15,16 @@ use tokio::{
     task::JoinHandle,
 };
 
-use crate::{event, TransferDescriptor};
+use crate::{event, KeyStore, TransferDescriptor};
 
 pub type Result<T = ()> = std::result::Result<T, crate::LibdropError>;
 
 const SQLITE_TIMESTAMP_MIN: i64 = -210866760000;
 const SQLITE_TIMESTAMP_MAX: i64 = 253402300799;
 
+pub const PUBLIC_KEY_LENGTH: usize = 32;
+pub const SECRET_KEY_LENGTH: usize = 32;
```

Comment:
Those constants are available in `drop-auth/src/lib.rs` and you can use those. Defining those in two places can be error-prone